dune-project

```diff
@@ -1,4 +1,4 @@
-(lang dune 2.9.1)
+(lang dune 2.9)
```

Comment:
Is that change relevant here? Please add this in another commit if so (same below).

Reply:
Just saw the dune warning about it! Please create another commit for those changes then!